pydantic_ai_slim/pydantic_ai/result.py

```diff
@@ -408,6 +410,24 @@ async def stream_output(self, *, debounce_by: float | None = 0.1) -> AsyncIterat
         else:
             raise ValueError('No stream response or run result provided')  # pragma: no cover
 
+    def stream_output_sync(self, *, debounce_by: float | None = 0.1) -> Iterator[OutputDataT]:
```

Comment:
Let's add this note that's in the `agent.run_sync` docstring to all these methods:

```
This is a convenience method that wraps [`self.X`][pydantic_ai.result.StreamedRunResult.X] with `loop.run_until_complete(...)`.
        You therefore can't use this method inside async code or if there's an active event loop.
```